src/db/db2/V1-dataflow.sql

```diff
@@ -0,0 +1,165 @@
+create sequence hibernate_sequence start with 1 increment by 1;
+
+create table app_registration (
```

Comment:
Do you think we should switch to a specific case (probably upper case) in these DDL scripts? I remember we had some case sensitive issues in the past.

Reply:
We can't as some db's will break. In fact it was task/batch which never should've use uppercase.